Code/GraphMol/MolDraw2D/MolDraw2D.h

```diff
@@ -35,8 +35,8 @@ using RDGeom::Point2D;
 namespace RDKit {
 
 class DrawText;
```

Comment:
The changes in this file are a mix of clang-format and some rearranging to make the code more readable.
There are no substantive changes.